.devcontainer/initialize-command.sh

```diff
@@ -16,12 +16,5 @@
 
 set -e
 
-# create a docker network for morpheus
+# create a docker network for morpheus if it does not exist
 docker network inspect morpheus >/dev/null 2>&1 || docker network create morpheus
```

Comment:
Might be worth exploring whether this can be simplified using the Docker Compose integration with VS Code devcontainers: https://code.visualstudio.com/docs/devcontainers/create-dev-container#_use-docker-compose